xbmc/pvr/PVRManager.cpp

```diff
@@ -133,6 +140,19 @@ void CPVRManager::Announce(AnnouncementFlag flag, const char *sender, const char
     TriggerEpgsCreate();
     TriggerTimersUpdate();
   }
+  else if (strcmp(message, "OnSleep") == 0)
+  {
+    for (auto updateJob : m_pendingUpdates)
```

Comment:
https://github.com/xbmc/xbmc/blob/master/xbmc/pvr/PVRManager.h#L675

=> <code>std::vector\<CJob *> m_pendingUpdates;</code>

Wouldn't <code>auto * updateJob</code> resolve to <code>CJob ** updateJob</code>, then?